### DISCUSSION TITLE
Deploying `safetyGraphics` on a Shiny server.

### DISCUSSION THREAD

**Discussion by samussiah** [SafetyGraphics/safetyGraphics#508] [Q&A]
Related to #457.

I'm trying to deploy `safetyGraphics` on a Shiny server whose version of `rlang` is older than the version `safetyGraphics` uses or was installed with.  The issue is that `rlang`, `htmltools`, and `shiny` are all already loaded when the server runs the app and the app expects a newer version of `rlang`.

Is there a workaround here?  I've tried detaching and sourcing local installations of the problem packages but the app just spins.  Any advice would be most appreciated!

**Comment by xni7** [comment]
Your best bet might be [renv](https://rstudio.github.io/renv/articles/renv.html). It might not solve the issue if the `shiny` pkg is too old. Might be still worth a try.
